cartographer/mapping/internal/test_helpers.cc

```diff
@@ -120,6 +120,14 @@ proto::Trajectory* CreateTrajectoryIfNeeded(int trajectory_id,
   return trajectory;
 }
 
+proto::PoseGraph::Landmark CreateFakeLandmark(
```

Comment:
The function is located within the test namespace, that by itself should make their purpose apparent. Maybe "fake" can be thus removed from the name.

Reply:
fair enough. I was just being consistent with the rest of the functions. I can change it in a separate PR.